content/en/synthetics/_index.md

```diff
@@ -8,73 +8,49 @@ further_reading:
 - link: "https://www.datadoghq.com/blog/introducing-synthetic-monitoring/"
   tag: "Blog"
   text: "Introducing Datadog Synthetics"
-- link: "/synthetics/api_tests/"
+- link: "/synthetics/troubleshooting/"
   tag: "Documentation"
-  text: "Configure an API Test"
-- link: "/synthetics/browser_tests/"
+  text: "Troubleshooting Synthetics"
+- link: "/synthetics/settings/"
   tag: "Documentation"
-  text: "Configure a Browser Test"
-- link: "/synthetics/identify_synthetics_bots/"
-  tag: "Documentation"
-  text: "Identify Synthetics Bots"
+  text: "Synthetics Settings"
 ---
 
-## Overview
-
-Datadog Synthetics gives you a new layer of visibility on the Datadog platform. By monitoring your applications and API endpoints via simulated user requests and browser rendering, Synthetics helps you ensure uptime, identify regional issues, and track application performance. By unifying Synthetics with your metrics, traces, and logs, Datadog allows you to observe how all your systems are performing as experienced by your users.
-
 {{< img src="synthetics/synthetics_home_page.png" alt="Synthetics home page" >}}
```

Comment:
Let's use https://a.cl.ly/JruLgjxb instead 
Also, do you think we could remove the table of content on the right like it was done for RUM? So that we can preview a larger image?